src/lib/utils.js

```diff
@@ -294,17 +294,15 @@ function loadConfigFromPackageJson(filePath) {
  */
 function formatFailureMessage(testResult, config) {
   const rootPath = config.rootPath;
-  const useColor = config.useColor;
 
-  const colorize = useColor ? colors.colorize : function(str) { return str; };
   const ancestrySeparator = ' \u203A ';
-  const descBullet = colorize('\u25cf ', colors.BOLD);
+  const descBullet = chalk.bold('\u25cf ');
   const msgBullet = '  - ';
   const msgIndent = msgBullet.replace(/./g, ' ');
 
   if (testResult.testExecError) {
     const text = testResult.testExecError;
-    return descBullet + colorize('Runtime Error', colors.BOLD) + '\n' + text;
+    return descBullet + chalk.bold('Runtime Error') + '\n' + text;
```

Comment:
This part is also subtle different and I should check on how this works. Namely when we call this we could disable colors here previously but now we'll only disable if done globally. I don't know how important this actually was and never triggered it. This is also possible with chalk really easily (can create a new instance that is disabled), so that's what I would do here.